packages/react-scripts/layout/views/layout.ejs

```diff
@@ -42,9 +42,11 @@
 
     <script>
       window.SERVER_DATA = {};
-      <%- typeof cacheKey !== 'undefined' ? `SERVER_DATA.cacheKey = '${cacheKey}';` : ''; %>
-      SERVER_DATA.appPath = "<%- typeof appPathOverride !== 'undefined' ? appPathOverride : appPath('') %>";
+      <%= typeof cacheKey !== 'undefined' ? `SERVER_DATA.cacheKey = '${cacheKey}';` : ''; %>
+      SERVER_DATA.appPath = "<%= typeof appPathOverride !== 'undefined' ? appPathOverride : appPath('') %>";
+      SERVER_DATA.targetEnv = "<%= process.env.TARGET_ENV %>""
```

Comment:
does this right here mean that we'll know targetEnv on the client side? Could I use this in the Link component?

Reply:
Yes, but why would the Link component care which targetEnv you are?

Reply:
And it reminds me I need to add some warnings to the docs to use this value carefully

Reply:
when developing locally, we decided that if the Link component would take you to a different app within familysearch, that we would literally just redirect you to the app depending on the target env.

Reply:
oh ok, just for local dev. That makes sense